dspace-api/src/main/java/org/dspace/content/DSpaceObjectServiceImpl.java

```diff
@@ -25,8 +26,10 @@
 import org.dspace.content.authority.service.ChoiceAuthorityService;
 import org.dspace.content.authority.service.MetadataAuthorityService;
 import org.dspace.content.service.DSpaceObjectService;
+import org.dspace.content.service.ItemService;
```

Comment:
Because this is functionality related to mixing plain text and entities, this is resolved in https://github.com/atmire/DSpace/commit/a62a66ec412fb0d2ebb6b4dfdcd6dd5478f3860d part of https://github.com/DSpace/DSpace/pull/2331

Reply:
👍